cli/cmd/plugin/unmanaged-cluster/tanzu/tanzu.go

```diff
@@ -583,6 +588,24 @@ func parseTKRBom(fileName string) (*tkr.Bom, error) {
 	return bom, nil
 }
 
+func getAdditionalRepos(tkrBom *tkr.Bom, scConfig *config.UnmanagedClusterConfig) []string {
```

Comment:
@jpmcb idea to to increase simplicity.

What if the default (via cobra) for the additional repo was simply set to the result of https://github.com/jpmcb/community-edition/blob/5e818ef30d3fa0d9cd530663d5b216efa131d569/cli/cmd/plugin/unmanaged-cluster/tkr/tkr.go#L496-L498?

Wonder if that would make this logic go away? And also have a consistent source of truth, for example, when we generate the configuration file for he user, we could add the [tceRepo](https://github.com/jpmcb/community-edition/blob/5e818ef30d3fa0d9cd530663d5b216efa131d569/cli/cmd/plugin/unmanaged-cluster/tkr/tkr.go#L15) as a value, by default?

Reply:
I don't mind that idea at all - However, i will say one thing, cobra does _not_ do a great job of surfacing this and it would be a bit buried. So new-commers to the codebase may find it confusing where that default is actually coming from. If we're ok with this, I say let's do it.